client/src/components/Jobs/Jobs.jsx

```diff
@@ -7,8 +7,11 @@ import { formatDateTime } from 'utils/timeFormat';
 import JobsStatus from './JobsStatus';
 import './Jobs.scss';
 import * as ROUTES from '../../constants/routes';
+import Searchbar from '_common/Searchbar';
+import queryStringParser from 'query-string';
+import { useLocation } from 'react-router-dom';
 
-function JobsView({ showDetails, showFancyStatus, rowProps }) {
+function JobsView({ showDetails, showFancyStatus, rowProps, fromDashboard }) {
```

Comment:
> function JobsView({ showDetails, showFancyStatus, rowProps, fromDashboard }) {

Maybe we can make the `fromDashboard` param a bit more descriptive. Perhaps something like `includeSearchbar`?